### PR TITLE
README fix link: Customization, not Configuration

### PR DESCRIPTION
**Pull request by paulvi**
previous link was to nonexistent page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ In case this doesn't happen, navigate to:
 
 * [Icons Customization](https://github.com/vscode-icons/vscode-icons/wiki/Customization)
 * [Project Auto-Detection](https://github.com/vscode-icons/vscode-icons/wiki/Pad)
-* [Custom Configuration](https://github.com/vscode-icons/vscode-icons/wiki/Confguration)
+* [Customization](https://github.com/vscode-icons/vscode-icons/wiki/Customization)
```
Comment (JimiC) [inline README.md]: The correct page is https://github.com/vscode-icons/vscode-icons/wiki/Configuration.
The issue is a typo in the name not the link.

Comment (jens1o) [reply README.md]: good point

Comment (JimiC) [reply README.md]: No. `Custom Configuration` refers in configuring the `vscode` settings.

Comment (robertohuertasm) [reply README.md]: @paulvi, could you be so kind to update the link so we can merge this? Thank you very much!

**Pull request merged**